src/components/about/About.tsx

```diff
@@ -10,59 +9,54 @@ import Navbar from "@/components/shared/navbar/Navbar";
 import ThemeSelector from "@/components/theme-selector/ThemeSelector";
 import "./About.scss";
 
-type Props = RouteComponentProps<any>;
-
-@autobind
-class About extends React.Component<Props> {
-  public render() {
-    return (
-      <div className="about">
-        <Navbar
-          className="about__navbar"
-          title={i18n("nav_about")}
-          backButton={true}
-          onBackButtonClick={this.onNavbarBackButtonClick}
-        />
-
-        <Card className="about__app-preferences">
-          <div className="about__subtitle">{i18n("app_settings")}</div>
-
-          <div className="about__app-preferences__buttons">
-            <LocaleSelector />
-            <ThemeSelector />
-          </div>
-        </Card>
-
-        <Card className="about__about">
-          <div className="about__about__buttons">
-            <IconButton
-              link={i18n("contact_me_url")}
-              iconName="at"
-              text={i18n("contact_me")}
-            />
-
-            <IconButton
-              link="https://github.com/HorusGoul/atom-pwa"
-              iconName="source_branch"
-              text={i18n("source_code")}
-            />
-
-            <IconButton
-              link={`mailto:${i18n("author_email")}`}
-              iconName="bug_report"
-              text={i18n("bug_report")}
-            />
-          </div>
-        </Card>
-      </div>
-    );
-  }
-
-  private onNavbarBackButtonClick() {
-    const { history } = this.props;
-
-    history.push(MAIN_MENU);
-  }
+function About() {
+  const history = useHistory();
+
+  const onNavbarBackButtonClick = React.useCallback(
+    () => history.push(MAIN_MENU),
+    [history]
+  );
+  return (
+    <div className="about">
+      <Navbar
+        className="about__navbar"
+        title={i18n("nav_about")}
+        backButton={true}
+        onBackButtonClick={onNavbarBackButtonClick}
```

Comment:
Whist this is more like what was there before, for better accessibilitya back button that changes the route should probably be a link instead of an button with and `onClick`.

Reply:
I'm not sure I understood correctly, but AFAI remember inside `Button` component while rendering component that was used is `a` but based on link prop it decided whether to render it as button or as link with `href` prop.

Navbar component renders this button component,
https://github.com/OpenSourceRaidGuild/atom-pwa/blob/next/src/components/shared/button/Button.tsx#L27

I'm not sure if `href` prop will work in this case if we render it as a link. Let me know if I missed something

Reply:
To clarify, I'm suggesting refactoring the back button into a link instead of a button to improve accessibility. It's not needed in this PR or even in this raid though if you don't want to take it on now.

Reply:
Oh I see. Ya it'll be good to refactor that back button. I can create another issue for that and can submit it in another PR

Thanks for suggestion :)